orca-clouddriver/src/main/groovy/com/netflix/spinnaker/orca/clouddriver/DelegatingOortService.java

```diff
@@ -51,6 +51,11 @@ public Response getServerGroup(String app, String account, String region, String
   }
 
   @Override
+  public Response getServerGroup(String account, String serverGroup, String region) {
```

Comment:
Maybe flag the prior function with  `@Deprecated` as well here

Reply:
Good find. Missed that one. Made the change here: https://github.com/spinnaker/orca/pull/1680/files#diff-044981bc49bd67385ca8beec17533501R48

Reply:
Sweet let's merge